sycl/doc/extensions/IntelFPGA/FPGALsu.md

```diff
@@ -0,0 +1,112 @@
+
+# FPGA lsu
+
+The Intel FPGA `lsu` class is implemented in `CL/sycl/intel/fpga_lsu.hpp` which
+is included in `CL/sycl/intel/fpga_extensions.hpp`.
+
+The class `cl::sycl::intel::lsu` allows users to explicitly request that the
+implementation of a global memory access is configured in a certain way. The
+class has two member functions, `load()` and `store()` which allow loading from
+and storing to a `global_ptr`, respectively, and is templated on the following
+4 optional paremeters:
+
+1.  **`cl::sycl::intel::burst_coalesce<N>`, where `N` is 0 or 1**: request, to
+the extent possible, that a dynamic burst coalescer be implemented when `load`
+or `store` are called.
+2. **`cl::sycl::intel::cache<N>`, where `N` is an integer greater or equal to
+0**: request, to the extent possible, that a read-only cache of the specified
+size be implemented when when `load` is called. It is not allowed to use that
+parameter for `store`.
+3. **`cl::sycl::intel::dont_statically_coalesce<N>`, where `N` is 0 or 1**:
+request, to the extent possible, that `load` or `store` accesses, should not be
+statically coalesced with other memory accesses at compile time.
+4. **`cl::sycl::intel::prefetch<N>`, where `N` is 0 or 1**: request, to the
```

Comment:
If some parameters have boolean semantics, the user-facing API should take a `bool` not 0 or 1